lib/DirectorySync.php

```diff
@@ -20,14 +20,11 @@ class DirectorySync
      * @param null|string $before Directory ID to look before
      * @param null|string $after Directory ID to look after
      * @param null|string $organizationId Unique ID for an organization
-     * @param \WorkOS\Resource\Order $order The Order in which to paginate records
+     * @param Resource\Order $order The Order in which to paginate records
      *
      * @throws Exception\WorkOSException
      *
-     * @return array An array containing the following:
-     *      null|string Directory ID to use as before cursor
-     *      null|string Directory ID to use as after cursor
-     *      array \WorkOS\Resource\Directory instances
+     * @return array{?string, ?string, Resource\Directory[]} An array containing the Directory ID to use as before and after cursor, and an array of Directory instances
```

Comment:
This array shape notation is supported by static analysis tools (PHPStan, psalm) and IDE. https://phpstan.org/writing-php-code/phpdoc-types#array-shapes